### PR TITLE
Personal keymap updates

### PR DESCRIPTION
**Pull request by bghull**
Getting updates pushed to the main repo, been a while

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/eyeohdesigns/babyv/keymaps/bghull/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#define AUTO_SHIFT_TIMEOUT 135
```
Comment (drashna) [inline keyboards/eyeohdesigns/babyv/keymaps/bghull/config.h:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Also, this needs an include guard, eg `#pragma once`

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/eyeohdesigns/babyv/keymaps/bghull/keymap.c [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |OneSFT| Left | Down |Right | ` ~  |        CLMK |  +   |   4  |   5  |   6  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |OneCTL|OneALT| Tab  |   _  |OneWIN|        QWER |  =   |   1  |   2  |   3  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |                    |      |      |             |SpcFUN| Ctrl |                    |
+ * `-----------------------------------------------------------------------------------'
+ */
+[NAVNUM] = LAYOUT_2u(
+  KC_ENT,  KC_HOME, KC_UP,   KC_END,  _______, _______, RESET,      KC_PMNS, KC_7,    KC_8,   KC_9,     KC_0,
```
Comment (fauxpark) [inline keyboards/eyeohdesigns/babyv/keymaps/bghull/keymap.c:76]: `RESET` should be `QK_BOOT`

File: keyboards/planck/keymaps/bghull/keymap.c [hunk 3]
```diff
@@ -16,86 +16,86 @@
...
+ * |                    |      |      |             |SpcFUN| Ctrl |                    |
  * `-----------------------------------------------------------------------------------'
  */
-[_NUMPAD] = LAYOUT_planck_grid(
-  _______, KC_BTN2, KC_MS_U, KC_BTN1,  KC_WH_U, KC_LPRN, KC_RPRN,    KC_7,     KC_8,    KC_9,    KC_0, _______,
-  _______, KC_MS_L, KC_MS_D, KC_MS_R,  KC_WH_D, KC_LBRC, KC_RBRC,    KC_4,     KC_5,    KC_6, KC_PPLS, KC_MINS,
-  _______, KC_MUTE, KC_VOLD, KC_VOLU, KC_GRAVE, KC_UNDS,  KC_EQL,    KC_1,     KC_2,    KC_3, KC_PGUP, KC_BSLS,
-  _______, XXXXXXX,   QK_BOOT, _______,  _______, _______, _______, _______,  KC_PDOT, KC_HOME, KC_PGDN, KC_END
+[NAVNUM] = LAYOUT_planck_grid(
+  KC_ENT,  KC_HOME, KC_UP,   KC_END, _______,  _______, RESET,      KC_PMNS, KC_7,    KC_8,   KC_9,     KC_0,
```
Comment (fauxpark) [inline keyboards/planck/keymaps/bghull/keymap.c:77]: As above.

**Comment by keyboard-magpie** [comment]
Apologies I've missed the QK_BOOT change and reverted last commit.

**Pull request merged**